### PR TITLE
Consulting

### PR DESCRIPTION
**Pull request by lubiarz-oliwia**
This PR adds:
- consulting page; 
- consulting form page;
- updated content for FAQ page, Home Page, and How it Works page;
- consulting page added in the navbar, footer

### CODE REVIEW

File: pages/consulting-form/_input-check.html
Comment (klijakub) [inline pages/consulting-form/_input-check.html:1]: No checkboxes in the consulting form, we can remove it

File: pages/consulting-form/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Contact Us
+permalink: /consulting-form/
+slug: contact
```
Comment (klijakub) [inline pages/consulting-form/index.html]: `slug: consulting-form`

File: pages/consulting/_comparison.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        <div class="comparison-col-2">
+          <div class="txt-box">
+            <p class="p-title">{{feature.traditional_consulting_title}}</p>
+            <p class="p-txt" style="color: #BFBFBF;">{{feature.traditional_consulting_description}}</p>
+          </div>
+        </div>
+        <div class="comparison-col-3">
+          <div class="txt-box">
+            <p class="p-title">{{feature.gruntwork_consulting_title}}</p>
+            <p class="p-txt" style="color: #BFBFBF;">{{{{feature.gruntwork_consulting_description}}</p>
```
Comment (klijakub) [inline pages/consulting/_comparison.html]: Why 4 `{{{{` ?

**Review by klijakub — CHANGES_REQUESTED** [review]

**Review by klijakub — APPROVED** [review]

File: assets/js/consulting.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+
+   (function($) {
```
Comment (ArkuVonSymfon) [inline assets/js/consulting.js]: some weird tabs spacing

File: pages/consulting-form/_form.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    <div class="row">
+      <div class="col-xs-12 col-md-4">
+        <label for="contact-message"></label>
+      </div>
+      <div class="col-xs-12 col-md-8">
+        <div class="text-center" style="text-align:right;display: block;">
+          <br>
+          <button type="submit" id="submit-button" class="col-xs-8 btn btn-primary" ga-on="click"
+            ga-event-category="contact" ga-event-action="submit" style="
+">
```
Comment (ArkuVonSymfon) [inline pages/consulting-form/_form.html]: the same

**Review by ArkuVonSymfon — APPROVED** [review]
You can repair the spacing issues but after all LGTM

**Comment by lubiarz-oliwia** [comment]
outdated

**Pull request closed without merging**